about/src/main/java/dev/arkbuilders/components/about/About.kt

```diff
@@ -0,0 +1,303 @@
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+@Composable
+fun ArkAbout(
+    appName: String,
+    @DrawableRes
+    appLogoResId: Int,
+    versionName: String,
+    privacyPolicyUrl: String
+) {
+    val ctx = LocalContext.current
+
+    var btcDialogVisible by remember { mutableStateOf(false) }
+    var ethDialogVisible by remember { mutableStateOf(false) }
+
+    QRCryptoDialog(
+        visible = btcDialogVisible,
+        title = stringResource(R.string.about_donate_btc),
+        wallet = stringResource(R.string.about_btc_wallet),
+        fileName = "ArkQrBtc.jpg",
+        qrBitmap = R.drawable.qr_btc
+    ) {
+        btcDialogVisible = false
+    }
+
+    QRCryptoDialog(
+        visible = ethDialogVisible,
+        title = stringResource(R.string.about_donate_eth),
+        wallet = stringResource(R.string.about_eth_wallet),
+        fileName = "ArkQrEth.jpg",
+        qrBitmap = R.drawable.qr_eth
+    ) {
+        ethDialogVisible = false
+    }
+
+    Column(
+        modifier = Modifier
+            .verticalScroll(rememberScrollState())
+    ) {
+        Column(
+            modifier = Modifier.padding(horizontal = 16.dp),
+            horizontalAlignment = Alignment.CenterHorizontally,
+        ) {
+            Icon(
+                modifier = Modifier.padding(top = 32.dp),
+                painter = painterResource(id = appLogoResId),
+                contentDescription = "",
+                tint = Color.Unspecified
+            )
+            Text(
+                modifier = Modifier.padding(top = 20.dp),
+                text = appName,
+                fontSize = 20.sp,
+                fontWeight = FontWeight.SemiBold,
+                color = ArkColor.TextPrimary
+            )
+            Text(
+                modifier = Modifier.padding(top = 12.dp),
+                text = stringResource(R.string.version, versionName),
+                color = ArkColor.TextTertiary
+            )
+            Text(
+                modifier = Modifier.padding(top = 12.dp),
+                text = "ARK Builders · Copyright ©2024",
+                color = ArkColor.TextTertiary
+            )
+            Row(
+                modifier = Modifier.padding(top = 8.dp),
+                verticalAlignment = Alignment.CenterVertically
+            ) {
+                SocialLink(
+                    painterResource(R.drawable.ic_about_site),
+                    text = "Website"
```

Comment:
I guess the text here should be put to `values/strings.xml` for future localization.
Same for following `SocialLink` texts.

Reply:
I don't think we should translate app names like Telegram and Discord